htlcswitch/resolve.go

```diff
@@ -0,0 +1,187 @@
+package htlcswitch
+
+import (
+	"github.com/go-errors/errors"
+	"github.com/jessevdk/go-flags"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"os"
+	"path/filepath"
+	"time"
+
+	"encoding/hex"
+	pb "github.com/ExchangeUnion/swap-resolver/swapresolver"
+)
+
+const (
+	defaultConfigFilename = "resolve.conf"
+	defaultServerAddress  = "127.0.0.1:10000"
```

Comment:
I don't know if it's worth choosing a default port other than 10000 to reduce the chance of the port being used. 10000 is used most notably for the Network Data Management Protocol.

Port 10011 is close and easy to remember and isn't used for anything significant that I can see.

Reply:
changed to 8886